src/main.cpp

```diff
@@ -2335,12 +2338,16 @@ void static FlushBlockFile(bool fFinalize = false)
 
 bool FindUndoPos(CValidationState &state, int nFile, CDiskBlockPos &pos, unsigned int nAddSize);
 
-static CCheckQueue<CScriptCheck> scriptcheckqueue(128);
 
-void ThreadScriptCheck() {
-    RenameThread("bitcoin-scriptch");
-    scriptcheckqueue.Thread();
-}
+
+// BU: moved to parallel.h
+//static CCheckQueue<CScriptCheck> scriptcheckqueue(128);
+
+//void ThreadScriptCheck() {
+//    RenameThread("bitcoin-scriptch");
+//    scriptcheckqueue.Thread();
+//}
```

Comment:
fixed...we don't need out old comments anymore since we're not rebasing on Core anymore.